### DISCUSSION TITLE
[Question] Does parcel generate different bundles with every run?

### DISCUSSION THREAD

**Discussion by sabatale** [parcel-bundler/parcel#7618] [Q&A] [answered]
Mozilla is checking one of our extensions that was bundled with Parcel for Firefox.

One of the requirements is to provide them with both the public archive and the source code - they want to bundle it themselves and then compare the two results. However, when they generate the parcel bundle, they end up with a different dist than the one provided to them.

Are we correct to assume generate slightly (or very) different dists with every run?

And therefore it's not possible for them to ever compare the two?

Our npm parcel configuration is the following:

```
  "scripts": {
    "build:content": "parcel build src/contentScript/contentScript.js --no-content-hash --no-source-maps --no-cache --detailed-report 0",
    "build:bg": "parcel build src/bgScript/bgScript.js --no-content-hash --no-source-maps --no-cache --detailed-report 0"
  }
```

**Comment by devongovett** [comment, marked as answer]
No, it should return the same code each time. If not, that's a bug. If you have a reproduction, you could file an issue.
